src/dftbp/dftbplus/initprogram.F90

```diff
@@ -1434,11 +1434,7 @@ subroutine initProgramVariables(this, input, env)
       call this%tblite%getOrbitalInfo(this%species0, input%slako%orb)
       allocate(input%slako%skOcc(input%slako%orb%mShell, input%geom%nSpecies))
       call this%tblite%getReferenceN0(this%species0, input%slako%skOcc)
-      ! Workaround: ifort 2021.7
```

Comment:
Intel 2021 is still listed under [Tested build environments](https://github.com/dftbplus/dftbplus/blob/main/INSTALL.rst#L114C1-L114C26). Is that supposed to be dropped for release 25.1?

Reply:
I guess we need to update that, probably job for a separate PR to match currently tested environments.